src/main/java/AppConfig.java

```diff
@@ -0,0 +1,46 @@
+import bank.*;
+import bank.clerk.*;
+import person.Person;
+
+public class AppConfig {
+
+    private final CentralBank centralBank = new CentralBank();
+    private final BankStorage bankStorage = new BankStorage(centralBank);
+    private final BankSystem bankSystem = new BankSystem(bankStorage);
+
+    public Person person(){
+         return new Person(bankKiosk());
+    }
+    public BankKiosk bankKiosk(){
+        return new BankKiosk(bankClerkFacade());
+    }
+    public BankServiceMediator bankClerkFacade(){
+        return new BankServiceMediator(bankClerk(), depositClerk(), withdrawClerk(), remittanceClerk(), changeStatusClerk(), getAccountInfoClerk());
+    }
+
+    public CreateAccountClerk bankClerk() {
+        return new CreateAccountClerk(bankSystem);
+    }
```

Comment:
```suggestion
    }
    
```
사소하지만 가독성을 위해 중요합니다!